lib/classes/account_class.rb

```diff
@@ -0,0 +1,36 @@
+module BankAccount
+
+  class Account
+    # changed to a reader since it doesn't seem like these attributes should be able to be overwritten
+    attr_reader :id, :initial_balance, :current_balance
+
+    def initialize(id, initial_balance)
+      raise ArgumentError.new ("You cannot create an account with a negative balance") \
+        unless initial_balance > 0
```

Comment:
If you need to wrap onto a second line I'd just go ahead and use the full unless/end structure rather than the postfix unless, as I think it's clearer.